data/students.json

```diff
@@ -46,5 +46,13 @@
     "linkedin_url": "https://www.linkedin.com/in/coraline-bodnar-660821113/",
     "email_gravatar": "coraline.bodnar@gmail.com",
     "team": "Purple"
+  },
+  {
+    "fullname": "Rosalie Leconte",   
+    "twitter": "Roza_lec",
+    "github": "Rosalieleconte",
+    "linkedin_url": "https://www.linkedin.com/in/rosalie-leconte-b75653b7/",
+    "email_gravatar": "leconte.rosalie@hotmail.fr",
+    "team": "Green"
```

Comment:
Can you add 

+  {
 +    "fullname": "Anthony Lefebvre",
 +    "twitter": "lefebvreant",
 +    "github": "lefebvreant",
 +    "linkedin_url": "https://www.linkedin.com/in/lefebvreant",
 +    "email_gravatar": "anthony.59.lefebvre@gmail.com",
 +    "team": "Green"
 +  },